src/app/lab/components/scenario-toggle.component.ts

```diff
@@ -49,7 +44,7 @@ export class ScenarioDropdownComponent implements OnInit {
 
     private getScenarios() {
         this.scenarioService.list().subscribe(data => {
-            this.scenarios = data;
+            this.scenarios = data.filter(s => s.name === 'RCP85' || s.name === 'RCP45')
```

Comment:
In the interest of constants book-keeping, consider saving constants 'RCP85' and 'RCP45' to a list and doing a `.includes(s.name)` or other check